artemis-server/src/main/java/org/apache/activemq/artemis/core/management/impl/ActiveMQServerControlImpl.java

```diff
@@ -2954,7 +2955,13 @@ public void onNotification(org.apache.activemq.artemis.core.server.management.No
       if (!(notification.getType() instanceof CoreNotificationType))
          return;
       CoreNotificationType type = (CoreNotificationType) notification.getType();
-      TypedProperties prop = notification.getProperties();
+      if (type == CoreNotificationType.SESSION_CREATED) {
+         TypedProperties props = notification.getProperties();
+         //SESSION_CREATED notification should not be broadcast twice
+         if (props.getIntProperty(ManagementHelper.HDR_DISTANCE) > 0) {
```

Comment:
Why would this logic specific to just SESSION_CREATED, would this not be generic to all?

Reply:
Also isnt cyclic between nodes in cluster already avoided, because of filter in the cluster connection bridge where it does a HDR_DISTANCE < flow.maxHops.

Reply:
To avoid calling the broadcast twice.The original logical SESSION_CREATED notification will not be transmitted between cluster nodes, and it is now required to transfer between cluster nodes. In order to maintain consistency with the original logic, this code to prevent the node receiving the SESSION_CREATED notification from making a broadcast call again.
Keep the original logic for other notification messages not judged